locale/fi/LC_MESSAGES/django.po

```diff
@@ -3673,78 +3673,68 @@ msgstr ""
-msgid ""
-"\n"
-"            You can change your avatar icon\n"
-"            at <a href=\"http://gravatar.com\">gravatar.com</a>.\n"
-"            Use your e-mail address <strong>%(email)s</strong>\n"
-"            when registering your account.\n"
-"            "
-msgstr ""
-"\n"
-"Voit vaihtaa profiilikuvaasi <a href=\"http://gravatar.com\">gravatar.com-"
-"palvelussa</a>. Käytä palvelussa samaa sähköpostiosoitettasi, <strong>"
-"%(email)s</strong>."
+#: userprofile/templates/userprofile/profile.html:59
+msgid "Copy API Access Token"
+msgstr "Kopioi API-käyttöoikeustietue"
 
-#: userprofile/templates/userprofile/profile.html:74
-msgid ""
-"\n"
-"            To change your settings, such as <strong>e-mail</strong>\n"
-"            address or <strong>password</strong>, you must update\n"
-"            them on your identity provider.\n"
-"            "
-msgstr ""
-"\n"
-"Käyttäjätilisi tietoja, kuten <strong>sähköpostiosoitetta</strong> tai "
-"<strong>salasanaa</strong>, voit muokata kirjautumiseen käyttämässäsi "
-"palvelussa."
+#: userprofile/templates/userprofile/profile.html:61
+msgid "Copy"
+msgstr "kopioi"
 
-#: userprofile/templates/userprofile/profile.html:81
-msgid "Automatic service redirections"
-msgstr "Automaattiset uudelleenohjaukset palveluihin"
+#: userprofile/templates/userprofile/profile.html:66
+msgid "Automatically log in to:"
+msgstr "Kirjaudu automaattisesti sisään:"
 
-#: userprofile/templates/userprofile/profile.html:83
+#: userprofile/templates/userprofile/profile.html:68
 msgid ""
-"\n"
-"\t\t\t\tThese are all services that you have marked for automatic "
-"redirection on this device.\n"
-"\t\t\t\tYou can remove automatic redirection here and you will be shown the "
-"privacy information next time you are being redirected to the service.\n"
-"\t\t\t\t"
+"These are all services that you have marked for automatic redirection on "
+"this device. You can remove automatic redirection here and you will be shown "
+"the privacy information next time you are being redirected to the service."
 msgstr ""
-"\n"
-"\t\t\t\tOlen merkinnyt nämä kaikki palvelut automaattisesti "
-"uudelleenohjattaviksi tällä laitteella.\n"
-"\t\t\t\tVoit poistaa automaattisen uudelleenohjauksen. Sinulle näytetään "
-"yksityisyydensuojasta tieto ensikerralla kun sinut ollaan ohjaamassa "
-"palveluun.\n"
-"\t\t\t\t"
+"Olen merkinnyt nämä kaikki palvelut automaattisesti uudelleenohjattaviksi "
+"tällä laitteella.Voit poistaa automaattisen uudelleenohjauksen. Sinulle "
+"näytetään yksityisyydensuojasta tieto ensikerralla kun sinut ollaan "
+"ohjaamassa palveluun"
 
-#: userprofile/templates/userprofile/profile.html:88
+#: userprofile/templates/userprofile/profile.html:75
 msgid "Forget automatic redirection"
-msgstr "Unohda automaattinen uudelleenohjaus."
+msgstr "Unohda automaattinen uudelleenohjaus"
+
+#: userprofile/templates/userprofile/profile.html:104
+msgid "Information provided by %(university)s"
+msgstr "Tiedot tarjoaa %(university)s"
```

Comment:
`"Tiedot tarjoaa %(university)s"` sounds funny in Finnish.
`Instituutiolta %(university)s saadut tiedot`